samples/contrib/versioned-pipeline-ci-samples/kaggle-ci-sample/download_dataset/download_data.py

```diff
@@ -0,0 +1,31 @@
+"""
+step #1: download data from kaggle website, and push it to gs bucket
+"""
+
+def processAndUpload(
```

Comment:
nit: `process_and_upload` for a more consistent naming convention.

Reply:
Thanks, will modify.